sherpa/tests/test_fit_unit.py

```diff
@@ -1876,22 +1876,22 @@ def test_fit_single(stat, usestat, usesys, finalstat):
     assert fr.statval == pytest.approx(finalstat, rel=7e-5)
 
 
-@pytest.mark.parametrize("stat,usestat,usesys,finalstat", [
-    (LeastSq, False, False, fit_lsq),
-    (Chi2, True, True, fit_chi2_tt),
-    (Chi2Gehrels, True, True, fit_chi2_tt),
-    (Chi2Gehrels, False, False, fit_gehrels_ff),
-    (Chi2XspecVar, False, True, fit_xvar_ft),
-    (Chi2ConstVar, False, True, fit_cvar_ft),
-    (Chi2ModVar, True, True, fit_mvar_xt),
-    (Chi2ModVar, True, False, fit_mvar_xf),
-    (Chi2ModVar, False, False, fit_mvar_xf),
-    (Chi2ModVar, False, True, fit_mvar_xt),
-
-    (Cash, True, True, fit_cash),
-    (CStat, True, True, fit_cstat),
+@pytest.mark.parametrize("stat,usestat,usesys,finalstat,rel", [
+    (LeastSq, False, False, fit_lsq, 1e-6),
+    (Chi2, True, True, fit_chi2_tt, 1e-6),
+    (Chi2Gehrels, True, True, fit_chi2_tt, 1e-6),
+    (Chi2Gehrels, False, False, fit_gehrels_ff, 1e-6),
+    (Chi2XspecVar, False, True, fit_xvar_ft, 1e-6),
+    (Chi2ConstVar, False, True, fit_cvar_ft, 1e-6),
+    (Chi2ModVar, True, True, fit_mvar_xt, 1e-6),
+    (Chi2ModVar, True, False, fit_mvar_xf, 1e-6),
+    (Chi2ModVar, False, False, fit_mvar_xf, 1e-6),
+    (Chi2ModVar, False, True, fit_mvar_xt, 1e-6),
+
+    (Cash, True, True, fit_cash, 1e-6),
+    (CStat, True, True, fit_cstat, 2e-4),  # This tests gives different results on Linux and ARM
```

Comment:
I am honestly surprised that this is needed (given the other tests and our tests using CStat elsewhere), but the CI tests do show the difference. I don't have the energy to try and find out why (especially not when I do have a lot of internal changes to the optimization code waiting to be finished).